website/docs/basics/blocks.md

```diff
@@ -3,3 +3,50 @@ id: blocks
 title: Blocks
 sidebar_label: Blocks
 ---
+
+Since WordPress 5.0, we were introduced to the new editing experience using block. With blocks, the idea is to have the same look and feel in the editor as it is on the frontend. As a developer, you just got a new technology that you should know if you want to do a great project on WordPress. That new technology is React because everything regarding blocks is done using React. 
```

Comment:
```suggestion
Since WordPress 5.0, we were introduced to the new editing experience using so-called Gutenberg blocks. With blocks, the idea is to have the same look and feel in the editor as it is on the frontend. As a developer, you just got a new technology that you should know if you want to build a great project on WordPress. That new technology is React because everything regarding blocks is done using React. 
```